app/views/devise/sessions/new.html.haml

```diff
@@ -8,7 +8,12 @@
       .col-lg-12
         = f.input_field :password, placeholder: t(:password)
     - if devise_mapping.rememberable?
-      = f.input :remember_me, as: :boolean, label: false, inline_label: true
+      .form-group
+        .col-lg-12
+          .checkbox
+            %label
+              %input{type: "checkbox"}/
```

Comment:
Unfortunately, that will not work. It will display the checkbox on the page, but it will not pass any information to the server when the form is submitted.

We have to keep `= f.input :rembember_me ...` line. It will provide the checkbox with the correct label. But we need to wrap it with some extra CSS classes.  Very similar to that example

https://github.com/ck3g/homebugh/blob/5ecb501407dcd2106467af16e20f34c93504beb6/app/views/categories/_form.html.haml#L4-L7


FYI: The syntax `f.input` comes from the [`simple_form`](https://github.com/heartcombo/simple_form) gem, and provides all required settings for form input. Such as labels, names, and some wrapper classes.

Reply:
@ck3g Sorry for the delay, but I've made the changes. Please review and let me know if I did it right this time. And thank you so much for being so patient with me :( Also, I had questions about what I should rollback exactly or remove.

Reply:
Yes you did everything right. Thank you.

Could you please rollback (undo) the changes in the following files: `.ruby-version`, `.rvmrc`, and  `app/assets/stylesheets/custom.css.scss`. See ["Files changed"](https://github.com/ck3g/homebugh/pull/93/files) tab to see them.

The changes in `app/assets/stylesheets/custom.css.scss` conflicts with the current changes in the master branch, that doesn't allow to proceed with this pull request.

Reply:
So, I've cleaned up my commits a little bit, please let me know if this is okay.

Reply:
Perfect! Thanks a lot

Reply:
Thanks again @ck3g :) learnt a lot of cool things due to the PR!

Reply:
That's great! 😃